tests/params/simple_integration.yml

```diff
@@ -161,11 +165,31 @@ partnership:
           min: 1
           max: 10
   sex:
+    frequency:
+      Sex:
+        type: bins
+        bins:
+          1:
+            prob: 0.5
+            min: 10
+            max: 50
+          2:
+            prob: 1.0
+            min: 10
+            max: 37
     acquisition:
       MSM:
         insertive: 0.0011
         receptive: 0.0138
         versatile: 0.00745
+  injection:
+    frequency:
```

Comment:
wasn't injection frequency found not to do anything and num_acts is what we landed on? pretty sure I deleted this in my pr